compiler-plugin/src/main/java/io/ballerina/stdlib/graphql/compiler/Utils.java

```diff
@@ -243,4 +252,26 @@ public static boolean isContextParameter(TypeSymbol typeSymbol) {
     public static String getAccessor(ResourceMethodSymbol resourceMethodSymbol) {
         return resourceMethodSymbol.getName().orElse(null);
     }
+
+    public static boolean isGraphQLServiceObjectDeclaration(
+            ModuleVariableDeclarationNode moduleVariableDeclarationNode) {
```

Comment:
Let's update the formatting here.

Reply:
This line exceeds the max char limit if aligned with the bracket

Reply:
This is a subjective take on this, but I think the variable name is too long. We can use something like `variableNode` here, since it is clear in this context. We can reduce the characters that way. Note, the intention here is not to change the name for the sake of shortening the line.